android/src/main/kotlin/com/hypertrack/sdk/flutter/HyperTrackPlugin.kt

```diff
@@ -86,15 +89,14 @@ public class HyperTrackPlugin(): FlutterPlugin, MethodCallHandler, StreamHandler
       "isRunning" -> result.success(sdk.isRunning)
       "start" -> start(result, sdk)
       "stop" -> stop(result, sdk)
-      "addGeotag" -> addGeotag(call.arguments(), result, sdk)
+      "addGeotag" -> addGeotag(call.arguments()!!, result, sdk)
```

Comment:
this will crash if arguments are null

Reply:
Not possible as the arguments will never become null due to null safety introduction